src/enableDisableRules.ts

```diff
@@ -17,6 +17,7 @@
 
 import * as ts from "typescript";
 import * as Lint from "./lint";
```

Comment:
let's remove this line too and do an explicit import of `IEnableDisablePosition`